src/test/java/org/kohsuke/github/AppTest.java

```diff
@@ -861,10 +862,27 @@ public void notifications() throws Exception {
         for (GHThread t : gitHub.listNotifications().nonBlocking(true).read(true)) {
             if (!found) {
                 found = true;
+                // both thread an unread are included
+                assertThat(t.getTitle(), is("Create a Jenkinsfile for Librecores CI in mor1kx"));
+                assertThat(t.getLastReadAt(), notNullValue());
+                assertThat(t.isRead(), equalTo(true));
+
                 t.markAsRead(); // test this by calling it once on old notfication
             }
-            assertNotNull(t.getTitle());
-            assertNotNull(t.getReason());
+            assertThat(t.getReason(), oneOf("subscribed", "mention", "review_requested", "comment"));
+            assertThat(t.getTitle(), notNullValue());
+            assertThat(t.getLastCommentUrl(), notNullValue());
+            assertThat(t.getRepository(), notNullValue());
+            assertThat(t.getUpdatedAt(), notNullValue());
+            assertThat(t.getType(), oneOf("Issue", "PullRequest"));
+
+            // both thread an unread are included
+            // assertThat(t.getLastReadAt(), notNullValue());
+            // assertThat(t.isRead(), equalTo(true));
```

Comment:
Are these commented out intentionally?

Reply:
Yes, I meant this as a note to say, "This query doesn't return only records where `isRead()` is `true`."